provider/defaultingress/ingress.go

```diff
@@ -0,0 +1,453 @@
+package defaultingress
```

Comment:
Would you mind if we call it default application ingress?

Today we don't expose, but there's also an API ingress. So if we ever end up opening that it will be more complicated to adjust the terms :thinking:

Reply:
I agree that `ingress` might be better package name here
The `defaultingress` might confuse
@tsorya any thoughts here?

Reply:
As we are working only with default one I think we should stick with default in name. When we will have an option to work with not only default one we can change the package and add default to variables that can be set

Reply:
Ok, I'm fine with pushing like this
Anyway, the package name is internal detail and can be changed without any issue